package.json

```diff
@@ -24,7 +31,7 @@
     "eslint": "^8.11.0",
     "prettier": "^2.2.1",
     "typescript": "^4.1.5",
-    "parcel-bundler": "^1.12.4"
+    "parcel": "^2.7.0"
   },
```

Comment:
you can add 
`   "node": "^18.12.0",`
to use node 18 under the hood :) 
 (or simply run)
`npm install node@18 --save-dev`

Reply:
Oh, I'm sorry I missed that (even though that was really the point of my PR...to upgrade node :) ).
I was apparently just relying on the fact that I had a new node installation locally.
I'll make the change you suggest, and submit another change which updates package.json to ask for the higher/newer version of node. Certainly, that is a way to reduce a dependency.

Reply:
Actually maybe it is better not to include node itself in the dependencies. If we do that I think I'd want to do it in all the projects. But as I know from work with Alois last week, we take special care on our build machines to use a specific path to an installed node. If we start a habit of pulling a node from within node_modules, it might be confusing. (we have a version of node we run npm install with, then a different version for all subsequent commands?). So I'll just check this in as is.

Reply:
Not a big fan of including node as a dependency, that seems odd. Maybe this is an option though: https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json